perma_web/api/views.py

```diff
@@ -279,11 +280,19 @@ class LinkFilter(django_filters.rest_framework.FilterSet):
     date = django_filters.IsoDateTimeFilter(field_name="creation_timestamp", lookup_expr='date')      # ?date=
     min_date = django_filters.IsoDateTimeFilter(field_name="creation_timestamp", lookup_expr='gte')   # ?min_date=
     max_date = django_filters.IsoDateTimeFilter(field_name="creation_timestamp", lookup_expr='lte')   # ?max_date=
-    url = django_filters.CharFilter(field_name="submitted_url", lookup_expr='icontains')              # ?url=
+    url = django_filters.CharFilter(method='surt_filter')                                             # ?url=
+
     class Meta:
         model = Link
         fields = ['url', 'date', 'min_date', 'max_date']
 
+    def surt_filter(self, queryset, name, value):
```

Comment:
Rename `name` to `_` to hint that it's intentionally unused? (This is totally up to your personal style.)

Reply:
Sure, can do; I copied those var names straight from the docs and didn't think about it.